Dockerfile

```diff
@@ -10,50 +10,32 @@ RUN apt-get update  \
     python3-pip \
     python3-venv \
     nano \
-    gcc \
     python3-dev \
-    git \
     build-essential \
-    htop \
-    libgl1 \
-    libglib2.0-0 \
-    libpython3-dev \
-    gnupg \
-    g++ \
-    libusb-1.0-0 \
+    libvips-dev \
     && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*
 
-RUN git clone https://git.ffmpeg.org/ffmpeg.git
```

Comment:
why did the ffmpeg go away? we do the defisheye in another way?

Reply:
Yes we do it in python now with OpenCV